src/GraphQL.Tests/ObjectExtensionsTest.cs

```diff
@@ -72,6 +72,110 @@ public void convert_single()
 
             /* Then */
             actual.ShouldBe(value);
+        }
+        
+                [Fact]
+        public void convert_double_array_to_array()
+        {
+            // Arrange
+            var doubles = new double[] { 1.00, 2.01, 3.14 };
+
+            // Act 
+            var result = doubles.GetPropertyValue(typeof(double[]));
+
+            // Assert
+            Assert.Equal(result, doubles);
```

Comment:
This project uses `Shoudly` if you can please update these assertions to that.

`result.ShouldBe(doubles)`.